src/ordered.rs

```diff
@@ -59,6 +59,7 @@ pub enum Method {
 }
 
 #[cfg(feature = "std")]
+#[cfg_attr(target_family = "wasm", inline(never))]
 fn measure_n_runs(
     n_runs: u128,
```

Comment:
Another way to stop the infinite loop from happening. Just call a `console.log` inside here printing the value of `n_runs`. The compiler will be unable to optimize the function. Resulting in it working properly.

Reply:
inline never is good enough I would think, but it needs a comment as to why it is required would be welcome